### PR TITLE
[Keyboard] Updates ScrabblePad, try 2

### PR DESCRIPTION
**Pull request by DonutCables**
Basically a copy of #12894 , but I ruined that one when I did a reset yesterday without realizing that one from a month ago wasn't actually pushed.

Updates to ScrabblePad files to facilitate VIA compatibility including:    

- Cleanup of core files

- Addition of VIA keymap folder and files    

- Update to vendor and board info to align with VIA requirements

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/donutcables/scrabblepad/rules.mk [hunk 1]
```diff
@@ -12,19 +12,10 @@ MCU = at90usb1286
...
-UNICODE_ENABLE = no         # Unicode
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-AUDIO_ENABLE = no           # Audio output on port C6
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = yes
+COMMAND_ENABLE = yes
+NKRO_ENABLE = yes
+BACKLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/donutcables/scrabblepad/rules.mk:21]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/donutcables/scrabblepad/rules.mk [hunk 2]
```diff
@@ -12,19 +12,10 @@ MCU = at90usb1286
 BOOTLOADER = halfkay
```
Comment (fauxpark) [inline keyboards/donutcables/scrabblepad/rules.mk:12]: Remove the list above this line (I can't reach it).

File: keyboards/donutcables/scrabblepad/keymaps/via/rules.mk [hunk 3]
```diff
@@ -0,0 +1,8 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+BLUETOOTH_ENABLE = no
+LEADER_ENABLE = no
+MIDI_ENABLE = no
+KEY_LOCK_ENABLE = no
+TERMINAL_ENABLE = no
```
Comment (fauxpark) [inline keyboards/donutcables/scrabblepad/keymaps/via/rules.mk:8]:
```suggestion
```

File: keyboards/donutcables/scrabblepad/keymaps/via/config.h [hunk 4]
```diff
@@ -0,0 +1,18 @@
+/*
```
Comment (fauxpark) [inline keyboards/donutcables/scrabblepad/keymaps/via/config.h:1]: This file can be deleted.

File: keyboards/donutcables/scrabblepad/config.h [hunk 5]
```diff
@@ -19,167 +19,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
-/* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
-#define DEVICE_VER      0x0001
-#define MANUFACTURER    Donut Cables
+#define VENDOR_ID       0x4443
```
Comment (fauxpark) [inline keyboards/donutcables/scrabblepad/config.h:22]: Please keep the comments in this file as per the template.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**